yoonit-camera/src/main/java/ai/cyberlabs/yoonit/camera/analyzers/face/FaceAnalyzer.kt

```diff
@@ -91,68 +87,70 @@ class FaceAnalyzer(
                 // Used to crop the face image.
                 val closestFace: Face? = this.faceCoordinatesController.getClosestFace(faces)
 
-                // Transform the camera face coordinates to UI graphic coordinates.
-                // Used to draw the face detection box.
-                val detectionBox = this.faceCoordinatesController.getDetectionBox(
-                    closestFace,
-                    image
-                )
-
-                // Verify if has error on the closestFace and detectionBox.
-                if (this.hasError(closestFace, detectionBox)) {
-                    return@addOnSuccessListener
-                }
-
-                // Transform the camera face contour points to UI graphic coordinates.
-                // Used to draw the face contours.
-                val faceContours = this.faceCoordinatesController.getFaceContours(closestFace, image)
+                closestFace?.let {
+                    // Transform the camera face coordinates to UI graphic coordinates.
+                    // Used to draw the face detection box.
+                    val detectionBox = this.faceCoordinatesController.getDetectionBox(
+                            closestFace,
+                            image
+                    )
+
+                    // Verify if has error on the closestFace and detectionBox.
+                    if (this.hasError(detectionBox)) {
+                        return@addOnSuccessListener
+                    }
 
-                // Get face bitmap.
-                val faceBitmap: Bitmap = this.getFaceBitmap(
-                    mediaImage,
-                    closestFace!!,
-                    cameraRotation.toFloat()
-                )
+                    // Transform the camera face contour points to UI graphic coordinates.
+                    // Used to draw the face contours.
+                    val faceContours = this.faceCoordinatesController.getFaceContours(closestFace, image)
```

Comment:
Do not need to create the entire `faceCounters` if `CaptureOptions.faceCounters == false`, right?